test/Model.js

```diff
@@ -1133,9 +1133,12 @@ describe('Model', function () {
     });
   });
 
-  it('Get missing item', (done) => {
-
+  it('Delete item with timestamp rangeKey', async () => {
```

Comment:
Are we sure this test fails without the changes you made?

Reply:
Yes. That's why I always make first commit with tests only - it can be checked out to compare the effect of code changes on added tests.